test/e2e/density.go

```diff
@@ -150,11 +154,24 @@ var _ = Describe("Density", func() {
 			}
 			Logf("Found %d events", current)
 
-			// Verify there were no pod killings or failures
-			By("Verifying there were no pod killings or failures")
+			// Verify there were no failures.
+			hist := make(map[string]map[string]int)
```

Comment:
Can you please add a comment about keys? 

Nit: maybe a full name "history" would be more readable?